tests/conftest.py

```
@@ -476,3 +476,8 @@ def mock_encryption_helper():
 		mock_helper.encrypt = mock_instance.encrypt
 		mock_helper.decrypt = mock_instance.decrypt
 		yield mock_helper
+
+@pytest.fixture
+def sample_token_id():
```

Ruff (0.11.9)

Redefinition of unused `sample_token_id` from line 353

(F811)